src/igvfd/schemas/rodent_donor.json

```diff
@@ -99,6 +99,20 @@
             "enum": [
                 "Mus musculus"
             ]
+        },
+        "traits": {
+            "title": "Phenotypic traits",
+            "description": "A list of known phenotypic traits of the Rodent donor",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Phenotype term",
```

Comment:
probably phenotypic trait

Reply:
missed that, but @hitz pointed out that we should probably not define traits in three places, instead define it in donor only, and you can update the description in children schemas (like rodent donor)